include/zephyr/bluetooth/hci_vs.h

```diff
@@ -194,23 +194,29 @@ struct bt_hci_rp_vs_read_usb_transport_mode {
 	uint8_t  supported_mode[0];
 } __packed;
 
-#define BT_HCI_VS_USB_H2_MODE                  0x00
-#define BT_HCI_VS_USB_H4_MODE                  0x01
+#define BT_HCI_VS_USB_H2_MODE                   0x00
+#define BT_HCI_VS_USB_H4_MODE                   0x01
 
-#define BT_HCI_OP_VS_SET_USB_TRANSPORT_MODE    BT_OP(BT_OGF_VS, 0x0011)
+#define BT_HCI_OP_VS_SET_USB_TRANSPORT_MODE     BT_OP(BT_OGF_VS, 0x0011)
 
 struct bt_hci_cp_vs_set_usb_transport_mode {
 	uint8_t  mode;
 } __packed;
 
-#define BT_HCI_OP_VS_SET_MIN_NUM_USED_CHANS    BT_OP(BT_OGF_VS, 0x0012)
+#define BT_HCI_OP_VS_SET_MIN_NUM_USED_CHANS     BT_OP(BT_OGF_VS, 0x0012)
 
 struct bt_hci_cp_vs_set_min_num_used_chans {
 	uint16_t handle;
 	uint8_t  phys;
 	uint8_t  min_used_chans;
 } __packed;
 
+#define BT_HCI_OP_VS_SET_SCAN_REQ_RCVD_MASK     BT_OP(BT_OGF_VS, 0x0013)
+
+struct bt_hci_cp_vs_set_scan_req_rcvd_mask {
+	uint8_t  enable;
+} __packed;
+
```

Comment:
isnt this already here and documented here: https://github.com/zephyrproject-rtos/zephyr/blob/main/doc/connectivity/bluetooth/api/hci.txt#L763-L803

Reply:
Thank you!